source/developer-guides/debug.rst

```diff
@@ -0,0 +1,197 @@
+.. _debug:
+
+Source Level Debugging
+--------------
+
+This explains how to perform basic debugging operation with Intel UDK Debugger on SBL over serial port. (used Apollo Lake (ABL) as example)
```

Comment:
extend underline all the way wider than the header above.

Reply:
fixed